Confus/Confus/PlayerHandler.cpp

```diff
@@ -62,26 +63,43 @@ namespace Confus
 			playerPair.second.Player->update();
 			playerPair.second.AudioEmitter->updatePosition();
 		}
+        m_PlayerController->update();
+	}
+
+    ConfusShared::Player* PlayerHandler::getPlayer(const long long & a_PlayerId)
+	{
+        try
+        {
+            auto& pair = m_Players.at(a_PlayerId);
```

Comment:
Again not sure if added by you, but I suggest using find rather than catching an exception. If the player id is possibly not in there, you're most likely not wanting to use at. With find you can check if it was in the map or not